Improve doc for docker development. (#2276)
GEOSX/GEOSX: src/docs/sphinx/developerGuide/Contributing/InstallWin.rst

.. code:: shell

>     PS> $env:VERSION='212-910'

Reviewer: ```suggestion
    PS> $env:VERSION='213-913'
```

according to https://hub.docker.com/layers/geosx/ubuntu20.04-gcc9/213-913/images/sha256-4e20100e6a333a098a635dec3226380ca5f9131501723563090a4dc599582e63?context=explore

Author: PR `213` is still not merged, and if/when it is, it's not sure that it will be using built `913`.
I prefer to stick to current TPL tag `212-910` https://github.com/GEOSX/GEOSX/blob/develop/.travis.yml#L10 which is sure to have been working at least once! 🤣

Reviewer: Oups my bad, just read the newest ! sorry

    PS> $env:IMG='ubuntu20.04-gcc9'
    PS> $env:REMOTE_DEV_IMG="remote-dev-${env:IMG}"

Please note the preposition of ``env:`` in the windows formalisme. The ``${ORG}`` variable will be hard-coded as ``geosx``. The last variable will be use

> as the image name. ``212-910`` refers to a specific version of the TPLs which may not be up to date. Please refer to :ref:`Continuous_Integration_process` for further info.

Reviewer: ```suggestion
as the image name. ``213-913`` refers to a specific version of the TPLs which may not be up to date. Please refer to :ref:`Continuous_Integration_process` for further info.
```

    PS > docker ps -a
    CONTAINER ID   IMAGE                                 COMMAND               CREATED                  STATUS          PORTS                                     NAMES

>     1efffac66c4c   remote-dev-ubuntu20.04-gcc9:212-910   "/usr/sbin/sshd -D"   Less than a second ago   Up 18 seconds   0.0.0.0:64000->22/tcp, :::64000->22/tcp   remote-dev-ubuntu20.04-gcc9-212-910

Reviewer: ```suggestion
    1efffac66c4c   remote-dev-ubuntu20.04-gcc9:213-913   "/usr/sbin/sshd -D"   Less than a second ago   Up 18 seconds   0.0.0.0:64000->22/tcp, :::64000->22/tcp   remote-dev-ubuntu20.04-gcc9-213-913
```
